src/pytypes/onefuzztypes/models.py

```diff
@@ -440,6 +441,7 @@ class ProxyConfig(BaseModel):
 
 class ProxyHeartbeat(BaseModel):
     region: Region
+    proxy_id: UUID
```

Comment:
By making this field required, this will make parsing heartbeats from existing proxies during an upgrade fail.  This isn't an issue _here_, as we currently hard-shutdown proxies on upgrade, but is something to keep in mind in the future.

Reply:
Would you like me to change it then?